doc/source/cluster/cluster_under_construction/ray-clusters-on-vms/user-guides/launching-clusters/aws.md

```diff
@@ -0,0 +1,126 @@
+export AWS_ACCESS_KEY_ID=foo
+export AWS_SECRET_ACCESS_KEY=bar
+export AWS_SESSION_TOKEN=baz
+
+# alternatively, you can setup AWS credentials using ~/.aws/credentials file
+echo "[default]
+aws_access_key_id=foo
+aws_secret_access_key=bar
+aws_session_token=baz" >> ~/.aws/credentials
+```
+
+## Start Ray with the Ray Cluster Launcher
+
+Once Boto3 is configured to manage resources in your AWS account, you should be ready to launch your cluster using the cluster launcher. The provided [example-full.yaml](https://github.com/ray-project/ray/tree/master/python/ray/autoscaler/aws/example-full.yaml) cluster config file will create a small cluster with an m5.large head node (on-demand) configured to autoscale up to two m5.large workers ([spot-instances](https://aws.amazon.com/ec2/spot/)).
+
+Test that it works by running the following commands from your local machine:
+
+```
+# Download the example-full.yaml
+wget https://raw.githubusercontent.com/ray-project/ray/master/python/ray/autoscaler/aws/example-full.yaml
+
+# Create or update the cluster. When the command finishes, it will print
+# out the command that can be used to SSH into the cluster head node.
+ray up example-full.yaml
+
+# Get a remote shell on the head node.
+ray attach example-full.yaml
+
+# Try running a Ray program.
+python -c 'import ray; ray.init()'
+exit
+
+# Tear down the cluster.
+ray down example-full.yaml
+```
+
+Congrats, you have started a Ray cluster on AWS!
+
+
+If you want to learn more about Ray Cluster Launcher. See this blog post for a [step by step guide](https://medium.com/distributed-computing-with-ray/a-step-by-step-guide-to-scaling-your-first-python-application-in-the-cloud-8761fe331ef1) to using the Ray Cluster Launcher.
+
+
+## AWS Configurations
+
+### Using Amazon EFS
+
+To utilize Amazon EFS in the Ray cluster, install some utilities and mount the EFS in `setup_commands`, as follows. Note that these instructions only work if you are using the Ray Cluster Launcher on AWS.
+
+```
+# Note You need to replace the {{FileSystemId}} to your own EFS ID before using the config.
+# You may also need to set correct SecurityGroupIds for the instances in the config file.
+
+setup_commands:
+    - sudo kill -9 `sudo lsof /var/lib/dpkg/lock-frontend | awk '{print $2}' | tail -n 1`;
+        sudo pkill -9 apt-get;
+        sudo pkill -9 dpkg;
+        sudo dpkg --configure -a;
+        sudo apt-get -y install binutils;
+        cd $HOME;
+        git clone https://github.com/aws/efs-utils;
+        cd $HOME/efs-utils;
+        ./build-deb.sh;
+        sudo apt-get -y install ./build/amazon-efs-utils*deb;
+        cd $HOME;
+        mkdir efs;
+        sudo mount -t efs {{FileSystemId}}:/ efs;
+        sudo chmod 777 efs;
+```
+
+### Accessing S3
+
+In various scenarios, worker nodes may need write access to the S3 bucket. E.g. Ray Tune has the option that worker nodes write distributed checkpoints to S3 instead of syncing back to the driver using rsync.
```

Comment:
```suggestion
In various scenarios, worker nodes may need write access to an S3 bucket, e.g., Ray Tune has an option to write checkpoints to S3 instead of syncing them directly back to the driver.
```